tests/testthat/test-renderer2-showSelected-same.R

```diff
@@ -25,8 +25,10 @@ test_that("redundant aes not saved to tsv", {
   geom.tsv <- Sys.glob(
     file.path("animint-htmltest", "geom1_point_points_*.tsv"))
   expect_equal(length(geom.tsv), 1)
-  expected.names <- sort(c("x", "y", ss.vec))
-  tsv.data <- read.table(geom.tsv, header=TRUE)
+  expected.names <- sort(c("x", "y", ss.vec,
+                           "alpha", "colour", "fill",
+                           "shape", "size", "stroke"))
```

Comment:
Hey @faizan-khan-iit can you please revert this commit? In this test case, the tsv file should definitely NOT contains alpha/colour/fill/shape/size/stroke columns.

If these columns are output in the built data from the new ggplot2, then we need to write code that analyzes which columns are constant, and then removes those columns and stores those data as params (to reduce tsv file sizes).